certbot/constants.py

```diff
@@ -30,7 +30,7 @@
     certname=None,
     dry_run=False,
     register_unsafely_without_email=False,
-    update_registration=False,
+    update_account=False,
```

Comment:
Same comment about keeping `update_registration` here.

Reply:
I think you forgot to delete this line

Reply:
(this line is still here)

Reply:
Agreed. I unresolved the comment and began running tests. Once they finish locally I will push. Thanks!